classes/Currency.php

```diff
@@ -184,15 +214,28 @@ public function __construct($id = null, $idLang = null, $idShop = null)
             if (empty($idLang)) {
                 $idLang = Context::getContext()->language->id;
             }
+
+            /*
+             * Both these fields used to be string, but they were turned into translatable
```

Comment:
```suggestion
             * Both fields are used to be string, but they were turned into translatable
```

Reply:
Ok but rather `Both fields used to be string`